packages/Sortable/src/components/Item/Item.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import PropTypes from "prop-types";
+import HandleIcon from "@paprika/icon/lib/DragHandle";
+import { Draggable } from "react-beautiful-dnd";
+import Button from "@paprika/button";
+import SortableContext from "../../SortableContext";
+import { itemStyles, itemIndexStyles, itemHandleStyles, itemBodyStyles, itemCloseStyles } from "./Item.styles";
+
+const propTypes = {
+  child: PropTypes.node.isRequired,
+  index: PropTypes.number.isRequired,
+};
+
+const Item = ({ child, index }) => {
+  const { hasIndexes, onRemove } = React.useContext(SortableContext);
+
+  function handleRemove() {
+    onRemove(index);
+  }
+
+  return (
+    <Draggable draggableId={`draggable-${child.props["data-drag-id"]}`} index={index}>
+      {(provided, snapshot) => {
+        return (
+          <li
+            {...provided.draggableProps}
+            {...provided.dragHandleProps}
+            css={itemStyles}
+            data-is-dragging={snapshot.isDragging ? true : undefined}
+            isDragging={snapshot.isDragging}
+            ref={provided.innerRef}
+          >
+            <div css={itemHandleStyles} {...{}}>
```

Comment:
This is gross.  But it seems necessary due to our Babel preset... I believe that the `transform-react-constant-elements` plugin conflicts with the `styled-components` plugin that utilizes the `css` prop.

https://github.com/styled-components/babel-plugin-styled-components/issues/221

And unless we configure our own plugins and skip the preset, it seems we either don't use the `css` prop in these kinds of situations... or we insert this weird (but easily searched + removed) hack.

https://stackoverflow.com/questions/33707189/is-there-an-option-to-disable-a-plugin-in-babel-when-using-a-preset

Reply:
cc @tristanjasper, @alexzherdev

Reply:
FYI the plugin is from storybook's babel config, not paprika's. So essentially this fix should only be necessary for storybook to run which makes it even more sucky. Let's add a comment for this?

Reply:
https://github.com/storybookjs/storybook/issues/6928#issuecomment-500933394 Is this a scape for this?

Reply:
oh, I forgot you also had issues with the `css` prop. It's great if just upgrading storybook fixes this 👍

Reply:
I will investigate whether an update for Storybook and/or styled-components will help.
![](https://media.giphy.com/media/NS7gPxeumewkWDOIxi/giphy.gif)

Reply:
Success! No more weird `{...{}}` needed!